.github/workflows/spread.yaml

```diff
@@ -1,10 +1,43 @@
 name: Spread
 on:
   pull_request:
+    paths-ignore:
+      - .github/**
+      - "!.github/workflows/spread.yaml" # Run on this file getting changed
+      - docs/**
+      - schema/**
+      - tests/** # Don't run if only tests changed, unless a spread test changed.
+      - "!tests/spread/**"
+      - .editorconfig
+      - .pre-commit-config.yaml
+      - .readthedocs.yaml
+      - "*.md" # Only top-level Markdown files
```

Comment:
I suspect these lists could contain more files/directories (ex. `tools`, `osv-scanner.toml`, etc.) but these are probably good first sets that can be tuned later.